test/extended/clusterup.sh

```diff
@@ -158,6 +158,21 @@ function os::test::extended::clusterup::hostdirs () {
     BASE_DIR="${base_dir}" os::test::extended::clusterup::internal::hostdirs
 }
 
+# Tests bringing up the service catalog and provisioning a template
+function os::test::extended::clusterup::service_catalog() {
+    os::cmd::expect_success "oc cluster up --service-catalog --version=latest"
+    os::test::extended::clusterup::verify_router_and_registry
+    os::test::extended::clusterup::verify_image_streams
+    os::cmd::expect_success "oc login -u system:admin"
+    os::cmd::expect_success "oc adm policy add-cluster-role-to-group system:openshift:templateservicebroker-client system:unauthenticated system:authenticated"
+    #oc::cmd::expect_success "oc adm policy add-role-to-user view developer -n openshift"
```

Comment:
I'd like to better understand the difference between (1) and (2), then.

and yes, i'd expect you can run the SAR when the add-to-role is performed, but instead of doing the add-to-role, we should be passing the correct groups on the broker api invocation's userinfo object, and then we don't need to do the add-to-role, and we properly validate that we're consuming the groups information.